doc/contributing.md

```diff
@@ -524,9 +524,9 @@ Tests also help us be confident that we won't break your code in the future.
 
 When writing tests, don't test everything that the GMT function already tests, such as
 the every unique combination arguments. An exception to this would be the most popular
-methods, such as `plot` and `basemap`. The highest priority for tests should be the
-Python-specific code, such as numpy, pandas, and xarray objects and the virtualfile
-mechanism.
+methods, such as :meth:`pygmt.Figure.plot` and :meth:`pygmt.Figure.basemap`.
+The highest priority for tests should be the Python-specific code, such as numpy,
+pandas, and xarray objects and the virtualfile mechanism.
```

Comment:
Please note that this file is written in Markdown (strictly speaking, [MyST](https://myst-parser.readthedocs.io/en/latest/index.html)), not reStructuredText, so the syntax is different.

Please see https://myst-parser.readthedocs.io/en/latest/index.html for the syntax of MyST, also see https://github.com/GenericMappingTools/pygmt/pull/941 for why decided to use Markdown instead of reStructuredText.